scripts/export-dos-labs.py

```diff
@@ -4,15 +4,16 @@
 For a DOS-type framework this will export details of all "user-research-studios" services.
 
 Usage:
-    scripts/export-dos-labs.py <stage> <api_token> <framework_slug>
+    scripts/export-dos-labs.py <stage> <framework_slug>
 """
 import itertools
 import sys
-sys.path.insert(0, '.')
+sys.path.insert(0, '.')  # noqa
```

Comment:
The #noqa isn't required for this flake8 error. It's excluded by the ignore of `E402` in the scripts dir here https://github.com/alphagov/digitalmarketplace-scripts/blob/master/.flake8#L12

I'm going to add a commit removing the #noqas for this error after this review.